MsCorePkg/Library/SecureBootKeyStoreLib/SecureBootKeyStoreLib.inf

```diff
@@ -0,0 +1,37 @@
+## @file - SecureBootKeyStoreLib.inf
+#  Copyright (C) Microsoft Corporation. All rights reserved.
+#  SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+
+[Defines]
+  INF_VERSION    = 0x00010005
+  BASE_NAME      = SecureBootKeyStoreLib

```

Comment:
Since this is a new file, I recommend using the helpful naming convention of including the module type in the instance name and `BASE_NAME` here.

```suggestion
  BASE_NAME      = BaseSecureBootKeyStoreLib

```

Reply:
Updated! Thank you